.github/workflows/swift.yaml

```diff
@@ -85,13 +85,13 @@ jobs:
       run: tuist test --path Samples
 
   package-tests:
-    runs-on: macos-latest
+    runs-on: macos-14
 
     steps:
     - uses: actions/checkout@v4
 
     - name: Switch Xcode
-      run: sudo xcode-select -s /Applications/Xcode_${{ env.XCODE_VERSION }}.app
+      run: sudo xcode-select -s /Applications/Xcode_16.1.app
```

Comment:
why are we pinning this for now again? some test fails on newer macos?

Reply:
The image for MacOS 14 doesn't have Xcode 16.4 so that's why I had to hard code this.